Assets/Scripts/UI/EnemyHealthBar.cs

```diff
@@ -0,0 +1,61 @@
+using Examen.Spawning.BossSpawning;
+using FishNet.Object;
+using MarkUlrich.Health;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Examen.UI
+{
+    public class EnemyHealthBar : NetworkBehaviour
+    {
+        [SerializeField] private bool _damage;
+        [SerializeField] private Slider _healthBar;
+
+        public HealthData EnemyHealthData { get; set; }
+        public Slider HealthSlider => _healthBar;
```

Comment:
```suggestion
        public Slider HealthBar => _healthBar;
```
please keep the variable names the same